doc/en/user/source/production/container.rst

```diff
@@ -54,6 +52,17 @@ For more information about JVM configuration, see the article `Performance tunin
     
    The above results (from a 16 GB laptop) amount to initial heap size of 256m, and a max heap size of around 4 GB (or around 1/4 of system memory).
    
+Enable the Marlin rasterizer
+----------------------------
+
+The Marlin rasterizer in Java 8 and getting better performance and scalability while rendering vector data in Java 8. 
+In order to enable it add the following among the JVM startup options::
+
+     -Xbootclasspath/a:$MARLIN_JAR 
+     -Dsun.java2d.renderer=org.marlin.pisces.MarlinRenderingEngine 
+
+where ``$MARLIN_JAR`` is the location of the ``marlin*.jar`` file located in the geoserver/WEB-INF/lib directory.
```

Comment:
I am not too bent out of shape it is not like this is a standalone jetty.

That is quite the trick with find to locate the marlin jar, I was not going  to be able to do something similar for start.bat and will need the specific version.